### ISSUE TITLE
Stop dbeaver from autocreating *'s when working with a block quote

### ISSUE THREAD

**Issue by coleman-rik** [dbeaver/dbeaver#23373]
dBeaver 24.0.2

whenever I start a block quote with:
/* 
and press enter, dBeaver automatically creates:
*
*/
and it automatically starts each subequent line with *.

how do I get dbeaver to stop this anoying behavior?

I want to be able to type /* followed by enter and have dbeaver do nothing except move my cursor to the next line.

thanks.

**Comment by E1izabeth** [comment]
Thank you for sharing your use case.
If you print `/*`, then press Enter and delete the star, it will stop adding the start on the new lines.

**Comment by coleman-rik** [comment]
thanks, but how do I set it so that it **never** does this?
I shouldn't have to delete a million stars.
thanks.

**Comment by E1izabeth** [comment]
1) Write `/*`
2) Press enter, dBeaver automatically creates:
      `*`
      `*/`
3) Remove star in the middle, left  it this way
      `/*`

      `*/`
4) Press any number of times Enter while the cursor is inside multiline comment - no additional stars appear

**Comment by coleman-rik** [comment]
yes, you've already mentioned that I can manually remove the unwanted, unrequested, unneeded characters that dBeaver is making.

what you haven't answered is a way to get dBeaver to stop making them in the first place.

if all I cared about was deleting unwanted autocreated characters, I didn't need to open an incident here on github.

**Comment by E1izabeth** [comment]
> what you haven't answered is a way to get dBeaver to stop making them in the first place.

There is no such way now. I provided you a workaround.
Your issue is open.
